src/renderer/components/reader/ReaderMenu.tsx

```diff
@@ -92,7 +116,7 @@ export class ReaderMenu extends React.Component<Props, State> {
                 toggleMenu={toggleMenu}
                 focusMenuButton={this.props.focusNaviguationMenu}
             />
-        );
+        </> );
```

Comment:
Could you explain the rationale for the addition of the wrapping React `<>` `</>` fragment?
(sorry if this is a stupid question, I just want to make sure I understand the reason, to educate myself)

Reply:
I juste forgot to remove it, I wanted to add something but I found an other way to do it

Reply:
Do you intend to remove it, or leave it?

Reply:
I removed it